pkgs/top-level/all-packages.nix

```diff
@@ -1429,6 +1429,11 @@ with pkgs;
     withWayland = true;
   };
 
+  shadps4 = callPackage ../by-name/sh/shadps4/package.nix {
+    # relies on std::sinf & co, which was broken in GCC until GCC 14: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=79700
+    stdenv = gcc14Stdenv;
+  };
```

Comment:
You *could* also pin this in the expression itself. Up to you though

Reply:
I think pinning here makes more sense, so it's still possible to override via `stdenv`. `shadps4.override { gcc14Stdenv = gcc15Stdenv; }` just *bothers* me. :sweat_smile: